### PR TITLE
ISSUE-3828 MySQL Driver Compalibility

### LINKED ISSUE #3828: A driver compatibility question
## Summary

If someone use [mysql python driver](https://github.com/mysql/mysql-connector-python) connect databend will have some errors:

1. mysql.connector.errors.InterfaceError: MySQL Version '0.1.0-d757471-simd(1.59.0-nightly-2022-01-10T04:17:51.060993521+00:00)' is not supported.
2. mysql.connector.errors.DatabaseError: 1105 (HY000): Code: 5, displayText = sql parser error: Expected equals sign or TO, found: 'utf8'.
3. mysql.connector.errors.DatabaseError: 1105 (HY000): Code: 5, displayText = sql parser error: Expected equals sign or TO, found: ..

For the first:

The driver is created in 2014, so the driver does not support MySQL if the Version is less than 4.1. This is a case-by-case design.

```python
elif version < (4 1):
    print("Err the version is ", version)
    raise errors.InterfaceError(
        "MySQL Version '{0}' is not supported.".format(server_version))
```

For the second:

The driver will execute query `SET NAMES utf8 COLLATE utf8_general_ci` and databend not support.

For the third:

The driver will execute query `SET @@session.autocommit = 1` and databend not support.

## How to fix

For the first:

Modify databend version format as year.x.x. Maybe like this `22.0.1`

For the second:

Modify databend parser, if the query has `SET NAME` and `SET @@` return ok.

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

[MySQL Python connector](https://github.com/mysql/mysql-connector-python) will send some queries that databend not supported now,

so the driver will not connect the databend.

* some test not fmt, modify them.
    tests/databend-test
    tests/helpers/client.py
    tests/suites/0_stateless/00_0000_dummy_select_py.py

## Changelog

- Bug Fix

## Related Issues

Fixes #3828 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: query/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "databend-query"
-version = "0.1.0"
+version = "22.1.0"
```
Comment (PsiACE) [inline query/Cargo.toml:3]: It is better to have a version specification, and then all crates in the repo should follow

Comment (TCeason) [reply query/Cargo.toml:3]: This is just a try. In my mind:

> For the first:
> 
> Modify databend version format as year.x.x. Maybe like this 22.0.1
> 
> Or
> 
> year.month.other

Comment (BohuTANG) [reply query/Cargo.toml:3]: It looks good to add a version prefix(like '22.') to `get_fuse_version`

**Review by sundy-li — COMMENTED** [review]
LGTM, better to  add a test in `databend-test` using this MySQL client to cover this issue.

**Comment by TCeason** [comment]
> LGTM, better to add a test in `databend-test` using this MySQL client to cover this issue.

Yep I will add the test.

**Comment by BohuTANG** [comment]
Let's merge first.
Please add the test in later PR if have the plan, @TCeason

**Pull request merged**